### DISCUSSION TITLE
Question à propos du mapping

### DISCUSSION THREAD

**Discussion by TheCrashKing** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#89] [Q&A]
Bonjour à tous ! 
J'espère poser la question au bon endroit, je suis en train de me plonger dans le mapping et l'apprentissage de Blender à mes heures perdues, et je voulais savoir s'il y avait une liste des mapping ou des modifications nécessaires ( pas l'habitude de Github et pas l'impression d'avoir vu quelque chose à propos du mapping ) histoire de me faire la main sur des choses qui au mieux pourrais servir.
Je pensais à par exemple "nettoyer" Cayo Perico des props d'armes et drogues qui me ferait un bon exercice et qui pourrais potentiellement être utile par la suite.
Merci d'avance pour le temps consacré à ma question !

**Comment by NariieL** [comment]
Coucou ! ❤️ Excuse-moi mais j'ai du mal à comprendre précisément ta question. Est-ce que tu nous demandes si on possède une liste de lieu, endroit qu'on aimerait travailler et que tu souhaiterais faire ?

**Reply by TheCrashKing** [reply]
Hello, ouai pour faire court c'est ça l'idée, comme je debute autant me faire la main sur quelque chose qui pourrait servir.

**Comment by NariieL** [comment]
Mmmmmh, je réfléchis à ça et je reviens vers toi soon !
